code/modules/projectiles/guns/projectile.dm

```diff
@@ -286,3 +286,31 @@
 		playsound(M, empty_sound, 100, 1)
 		return 0
 	return ..()
+
+/obj/item/weapon/gun/projectile/proc/RemoveAttach(var/mob/user)
+	if(silenced)
+		if(!user.is_holding_item(src))
+			..()
+			return
+		to_chat(user, "<span class='notice'>You unscrew [silenced] from [src].</span>")
+		user.put_in_hands(silenced)
+		silenced = 0
+		w_class = W_CLASS_SMALL
+		update_icon()
+		return 1
+	return 0
+
+/obj/item/weapon/gun/projectile/verb/force_removeAttach()
```

Comment:
why is this called `force`? it doesn't seem to be forcing anything

Reply:
the only reason I was lazy to remove 'force_' because you can't give procs and verbs a same name(which was short)